operations/payments.md

```diff
@@ -190,3 +190,272 @@ Adds a new alternative payment to a specified customer.
 #### Payment next action discriminator
 
 * `RedirectToUrl` - Redirect customer to a URL where they can complete their payment.
+
+## Get all payments
+
+Returns all payments. Note this operation uses [Pagination](../guidelines/pagination.md).
```

Comment:
I think we could add a fuller description than this. If it was simply 'get all payments' then there would be no parameters. In fact PaymentIds, BillIds and AccountingStates are **required** parameters, so as a minimum it should be something like "Returns all payments with specified accounting states belonging to specified bills". PaymentIds is required - is that correct, does that mean it only returns specified payments?

Reply:
If you can come up with something would be good, I can't come up with something good, since it's getting all the payments (I've corrected optional/required filters).